dsdl_parser/src/parse.rs

```diff
@@ -170,10 +176,14 @@ fn is_bin_digit(chr: u8) -> bool {
     chr == b'0' || chr == b'1'
 }
 
-fn allowed_in_decimal_number(chr: u8) -> bool {
+fn is_allowed_in_float(chr: u8) -> bool {
     is_digit(chr) || chr == b'.' || chr == b'E' || chr == b'e' || chr == b'-'
 }
 
+fn is_allowed_in_const(chr: u8) -> bool {
+    is_digit(chr) || is_uppercase_char(chr) || is_lowercase_char(chr) || chr == b'.' || chr == b'E' || chr == b'e' || chr == b'-'
```

Comment:
@kjetilkjeka Here and above, how about `+`? I just consulted with the specification and noticed that it is, indeed, a bit vague about the validity of the plus sign in a definition. Let me go and extend it accordingly. The existing Python compiler blindly `eval()`s whatever is provided in the definition, so it can accept the plus as well.

Reply:
Nice catch! I was actually just about to push v0.1.0 of the parser so the timing couldn't have been better.

I will fix it tomorrow before pushing the new parser version.